common/src/View/MapFrame.cpp

```diff
@@ -972,6 +972,10 @@ namespace TrenchBroom {
          * This is relatively expensive so only call it when the clipboard changes or e.g. the user tries to paste.
          */
         bool MapFrame::canPaste() const {
+            if (!hasTransitiveFocus(m_mapView)) {
```

Comment:
Nitpick, but any additional conditions we add here also need to trigger canPaste to re-run.

E.g. if you clear the clipboard (Ctrl+C in Visual Studio), focus something that's not the map view, like the TB Console, then copy, canPaste will return false (correctly). But then if you focus the map view, canPaste won't be re-evaluated, and the Paste menu item will be incorrectly disabled.

It may be too expensive to re-evaluate canPaste whenever focus changes though (IIRC it's quite slow, like around 100ms)

Reply:
I wonder why it's so expensive. The Qt documentation doesn't say anything about it.

Reply:
Maybe we should just get rid of this check and leave the menu item enabled? It gets checked on paste anyway.

Reply:
Yeah, let's do that.